index.bs

```diff
@@ -1130,6 +1156,392 @@ The following extended attributes are applicable to interfaces:
 </div>
 
 
+<h3 id="idl-mixins">Mixins</h3>
+
+A <dfn export>mixin</dfn> is a definition (matching <emu-nt><a href="#prod-Mixin">Mixin</a></emu-nt>)
+that declares state and behavior that can be [=included=] by one or more [=interfaces=],
+and that are exposed by objects that implement an [=interface=] that [=includes=] the [=mixin=].
+
+<pre class="webidl" class="syntax">
+    mixin identifier {
+      /* mixin_members... */
+    };
+</pre>
+
+Note: Mixins, much like [=partial interfaces=], are intended for use as a specification editorial aide,
+allowing a coherent set of functionalities to be grouped together,
+and included in multiple interfaces, possibly across documents.
+
+A [=mixin=] is a specification of a set of <dfn export lt="mixin member">mixin members</dfn>
+(matching <emu-nt><a href="#prod-MixinMembers">MixinMembers</a></emu-nt>),
+which are the [=constants=], [=regular operations=], [=regular attributes=],
+and an eventual [=stringifier=],
+that appear between the braces in the mixin declaration.
+
+These [=constants=], [=operations=], and [=attributes=]
+describe the behaviors that can be implemented by an object,
+as if they were specified on the [=interface=] that [=includes=] them.
+
+As with interfaces, the IDL for mixins can be split into multiple parts by using
+<dfn export>partial mixin</dfn> definitions
+(matching <emu-t>partial</emu-t> <emu-nt><a href="#prod-Mixin">Mixin</a></emu-nt>).
+The [=identifier=] of a partial mixin definition must be the same as the identifier of a mixin definition.
+All of the [=mixin members|members=] that appear on each of the partial mixin definitions
+are considered to be members of the mixin itself.
+
+<pre class="webidl" class="syntax">
+    mixin <mark>SomeMixin</mark> {
+      /* mixin_members... */
+    };
+
+    partial mixin <mark>SomeMixin</mark> {
+      /* mixin_members... */
+    };
+</pre>
+
+The order that members appear in has significance for property enumeration
+in the <a href="#es-namespaces">ECMAScript binding</a>.
+
+Note that unlike [=interfaces=] or [=dictionaries=], [=mixins=] do not create types.
+
+Of the extended attributes defined in this specification,
+only the [{{Exposed}}] and [{{SecureContext}}] extended attributes are applicable to mixins.
+
+An <dfn>includes statement</dfn> is a definition
```

Comment:
I don't think it's a definition. Maybe a statement is what is meant here? Although then you're just saying "statement" three times in the same sentence :)

Reply:
I just parroted the [text for "implements statements"](https://heycam.github.io/webidl/#idl-implements-statements.), here. Note the spec specifically defines "implements statements" as definitions in the [definition of definition](https://heycam.github.io/webidl/#dfn-definition).